src/t8_schemes/t8_default/t8_default_hex/t8_default_hex.cxx

```diff
@@ -639,6 +639,7 @@ t8_default_scheme_hex::element_is_valid (const t8_element_t *element) const
          && T8_QUAD_GET_TDIM ((const p8est_quadrant_t *) element) == 3;
 }
 
+#endif
 void
 t8_default_scheme_hex::element_to_string (const t8_element_t *elem, char *debug_string, const int string_size) const
 {
```

Comment:
Why isn't the assert a problem because the element_is_valid above is only available in debug mode? Build is working.

Reply:
Because the function in T8_ASSERT is only compiled in debug-mode.

Reply:
Ah i see, then its perfectly fine